### PR TITLE
[CWS] fix model used for constants when checking policies using the security agent

### PR DESCRIPTION
**Pull request by paulcacheux**
### What does this PR do?

https://github.com/DataDog/datadog-agent/pull/24680 introduced the capacity to use a security agent running on linux and check policies designed for windows. Sadly the constants used are not the correct ones, this PR fixes this.

### CODE REVIEW

File: pkg/security/probe/probe.go [hunk 1]
```diff
@@ -366,7 +366,7 @@ func (p *Probe) GetService(ev *model.Event) string {
 func (p *Probe) NewEvaluationSet(eventTypeEnabled map[eval.EventType]bool, ruleSetTagValues []string) (*rules.EvaluationSet, error) {
 	var ruleSetsToInclude []*rules.RuleSet
 	for _, ruleSetTagValue := range ruleSetTagValues {
-		ruleOpts, evalOpts := rules.NewEvalOpts(eventTypeEnabled)
+		ruleOpts, evalOpts := rules.NewBothOpts(eventTypeEnabled)
```
Comment (YoannGh) [inline pkg/security/probe/probe.go:369]: I'm not sure but aren't we always using the the Linux model here, even when running the windows probe?

Comment (paulcacheux) [reply pkg/security/probe/probe.go:369]: basically:
`pkg/security/secl/model` is the model of the running architecture, on windows this is the windows model
`pkg/security/seclwin/model` is always the windows model, even on linux

So in pkg/security/probe it's using secl/model so windows on windows, linux on linux

Comment (YoannGh) [reply pkg/security/probe/probe.go:369]: right, I forgot `pkg/security/secl/model` also contained the windows model, thanks!

**Review by YoannGh — APPROVED** [review]

**Pull request merged**